one-pager-maker/src/stories/Usertem.tsx

```diff
@@ -0,0 +1,28 @@
+import {useState} from "react";
+import {Menu, MenuItem} from "./Menu.tsx";
+import {useDetectClickOutside} from "react-detect-click-outside";
+
+interface UserItemProps {
+    userName: string;
+    onClick: (user: string) => void;
+}
+export const UserItem: React.FC<UserItemProps> = ({userName, onClick}) => {
+    const [openMenu, setOpenMenu] = useState(false);
+    const ref = useDetectClickOutside({
+        onTriggered: () => setOpenMenu(false)
+    })
+    return (
+        <span ref={ref}>
+            <button onClick={() => setOpenMenu(true)}>
+                ...
```

Comment:
It might be better to use another icon like Settings
![image](https://github.com/Greek-Academy/one-pager-maker/assets/35805963/0acd0b4d-96a0-4e90-86af-6b305b983924)

Reply:
Is it okay to add react-icons?

Reply:
ok

Reply:
thank you